### PR TITLE
Feature/e comerce template 

### PR DESCRIPTION
**Pull request by danieltolmon**

### CODE REVIEW

File: staff/daniel-toledo/e-comerce-template/index.html [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    <header class='header'>
+        <img class='logo' src="http://pngimage.net/wp-content/uploads/2018/06/logo-planete-png.png" alt="logo">
+        <h1 class='logo-text'>Planet-Commerce</h1>
+        <form class=search>
+            <input type="Search...">
+            <button>
+                    <i class="fa fa-search"></i>
+            </button>
+        </form>
+        <picture class="w3-xxlarge w3-text-white burguer">
```
Comment (maryammdot) [inline staff/daniel-toledo/e-comerce-template/index.html]: Here the picture tag would not be necessary

File: staff/daniel-toledo/e-comerce-template/style.css [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+        font-size: 20px;
+        text-decoration: none;
+    }
+
+   
+    main{
+    
+        padding-top:180px;
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr;
```
Comment (maryammdot) [inline staff/daniel-toledo/e-comerce-template/style.css]: You could use repeat(3, 1fr)

File: staff/daniel-toledo/precourse/calculadora/main.js [hunk 3]
```diff
@@ -1,5 +1,4 @@
```
Comment (maryammdot) [inline staff/daniel-toledo/precourse/calculadora/main.js:1]: WOW!!!! this file should not be committed in this branch! Try to fix it, deleting it from here and committing it from the precourse branch. 🤓

File: staff/daniel-toledo/e-comerce-template/index.html (hunk 1 above)
Comment (danieltolmon) [reply staff/daniel-toledo/e-comerce-template/index.html]: I changed by `<div>`, would it be better?

File: staff/daniel-toledo/e-comerce-template/style.css (hunk 2 above)
Comment (danieltolmon) [reply staff/daniel-toledo/e-comerce-template/style.css]: Done!

**Pull request merged**